src/collective/cover/static/js/layout_edit.js

```diff
@@ -290,41 +290,10 @@
           'top': '0',
           'width': '15px'
         });
-        elements = elements !== undefined ? elements : le.find('.' + column_class + ', .' + tile_class + ', .' + row_class);
```

Comment:
this file is not in the JS registry, we don't need to cook the registry in the upgrade step; anyway, I think is a good idea to add it in the registry to guarantee it will be always updated in production sites.

Reply:
Let's think about it when we rewrite code to use webpack; I'll remove the upgrade step to cook javascript now.

Reply:
Put ```?v=xxxx``` in the file call to avoid problems with caching.

Reply:
Sorry. I saw you registered js.